nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,55 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param fieldElement The class field element.
+   * @param nodeReceiver The receiver as a {@link Node}.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAccessNode(Element fieldElement, Node nodeReceiver) {
+    Tree receiver = null;
+    if (nodeReceiver != null) {
+      receiver = nodeReceiver.getTree();
+    }
+    return fromFieldAccess(fieldElement, receiver);
+  }
+
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param fieldElement The class field element.
+   * @param receiver The receiver as a {@link MemberSelectTree}.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAccessTree(Element fieldElement, MemberSelectTree receiver) {
```

Comment:
This method seems to be of limited usefulness.  I'd prefer to delete and just have callers use `fromFieldAccess` below, passing the correct receiver